VAMobile/src/screens/HealthScreen/SecureMessaging/ViewMessage/MessageCard.tsx

```diff
@@ -101,14 +104,15 @@ const MessageCard: FC<MessageCardProps> = ({ message }) => {
     }
   }
 
+  const navigateToReplyHelp = () => {
+    logAnalyticsEvent(Events.vama_sm_nonurgent())
+    navigation.navigate('ReplyHelp' as never)
```

Comment:
This `never` can be removed the same way the other ones were

Reply:
updated